spec/test-utils/mockEndpoints.ts

```diff
@@ -56,3 +58,32 @@ export function mockSetupCrossSigningRequests(): void {
         {},
     );
 }
+
+/**
+ * Returns 404 `M_NOT_FOUND` for GET `/room_keys/version` request until POST `room_keys/version` is called.
+ * Once the POST is done GET `/room_keys/version` will return the posted backup instead of 404
```

Comment:
```suggestion
 * Mock out requests to  `/room_keys/version`.
 *
 * Returns `404 M_NOT_FOUND` for GET requests until `POST room_keys/version` is called.
 * Once the POST is done, `GET /room_keys/version` will return the posted backup
 * instead of 404.
```

Reply:
updated